### DISCUSSION TITLE
Can we have a standalone version of nunit console 3.16.3?

### DISCUSSION THREAD

**Discussion by Somecoolname444333** [nunit/nunit-console#1377] [General]
We are in need of the new version as an executable as we have to test .net 8 application.

**Comment by OsirisTerje** [comment]
Can you check out 3.15.5 ?

We plan to abandon the 3.16.X and move all working PRs over to the 3.15.X track.  It will be the 3.15 that will be the base for all future 3.X versions.

**Reply by Somecoolname444333** [reply]
If I run 3.15.5 I get the message: System.ArgumentException : Unknown framework version 8.0
Should it already support .net 8.0? I installed SDK on the machine

**Reply by Somecoolname444333** [reply]
Are you planing to integrate 8.0 SDK in 3.15.X soon?

**Comment by OsirisTerje** [comment]
Please see the pre-release of 3.17 (which is based on 3.15.5).  
See issue #1374 for details.

**Reply by Somecoolname444333** [reply]
Thank you, I tried this version. Testcases that do not reference other Microsoft Assemblies seam to work now. But those that reference for example Microsoft.Extetions.Chaching.Memory do not find the assembly. Running the same Unittest DLL with dotnet test works.

![image](https://github.com/nunit/nunit-console/assets/121042335/be9f1143-e9b5-4e88-a0e3-9d5a8c5d5471)

**Reply by Somecoolname444333** [reply]
I found the workaround. Had to change the framework to : Microsoft.AspNetCore.App

**Comment by OsirisTerje** [comment]
Awesome!   So this can be closed ?
